packages/design-system-tailwind-preset/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metamask/design-system-tailwind-preset",
-  "version": "0.5.0",
+  "version": "0.6.0",
```

Comment:
Bumping minor version as we are adding functionality without any breaking changes